dotnet/src/webdriver/Cookie.cs

```diff
@@ -272,27 +274,28 @@ internal long? ExpirySeconds
         /// </summary>
         /// <param name="rawCookie">The Dictionary object containing the cookie parameters.</param>
         /// <returns>A <see cref="Cookie"/> object with the proper parameters set.</returns>
+        /// <exception cref="ArgumentNullException">If <paramref name="rawCookie"/> is null.</exception>
         public static Cookie FromDictionary(Dictionary<string, object> rawCookie)
         {
             if (rawCookie == null)
             {
                 throw new ArgumentNullException(nameof(rawCookie), "Dictionary cannot be null");
             }
 
-            string name = rawCookie["name"].ToString();
+            string name = rawCookie["name"].ToString()!;
```

Comment:
If `name` and `value` are null, that'll surface in the constructor for Cookie. We can just ignore it here.